app/src/main/java/com/x8bit/bitwarden/ui/platform/theme/Type.kt

```diff
@@ -252,4 +273,37 @@ data class NonMaterialTypography(
     val labelMediumProminent: TextStyle,
     val sensitiveInfoSmall: TextStyle,
     val sensitiveInfoMedium: TextStyle,
```

Comment:
Can we make sure the `sensitiveInfoSmall` and `sensitiveInfoMedium` get updated too

Reply:
https://www.figma.com/design/gAE9D9OP2zOuVU7aSiIbHy/Bitwarden-Android-Mobile-App-Reskin---V3-(current)?node-id=49848-6285&node-type=frame&t=h4SS0xWRUTWfNjMF-0

From this looks like they are remaining as they are currently.

Reply:
Ohh, nice!

I assumed they would be dm sans too

Reply:
DM Sans is just not as sensitive as a mr. roboto